src/types/stardust.d.ts

```diff
@@ -0,0 +1 @@
+declare module '@nebula.js/stardust';
```

Comment:
We should check with Thomas on the state of stardust types being exported.

Reply:
Agree. I just made dirty quick fix to remove the type error.